Microsoft.Azure.Cosmos/src/DocumentClient.cs

```diff
@@ -1396,6 +1421,8 @@ public void Dispose()
 
         internal virtual Task<QueryPartitionProvider> QueryPartitionProvider => this.queryPartitionProvider.Value;
 
+        public ClientTelemetry ClientTelemetryInstance { get; set; }
```

Comment:
```suggestion
        public ClientTelemetry ClientTelemetryInstance { get; private set; }
```

Shouldn't the setter be private?